### ISSUE TITLE
Исправление ошибки выгрузки игроков

### ISSUE THREAD

**Issue by ViacheslavMelnikov** [Studio-Yandex-Practicum/adaptive_hockey_federation#362]
Выгрузка игроков по поиску выдает ошибку по вновь созданному полю discipline_name.
Если в поиске по игрокам указать поле ВСЕ и в поле поиска указать к примеру искомое значения поля ДИСЦИПЛИНА - результат будет не правильным.
Необходимо для поиска и выгрузки использовать унифицированный алгоритм заключенный в функцию.

# Предложения и предположения как исправить
Ошибка в файле unloads/utils.py строка 17 
заменить на Q(discipline_name__name__icontains=search)

![image](https://github.com/Studio-Yandex-Practicum/adaptive_hockey_federation/assets/120438727/de346c9e-c783-4b09-837b-f91e32e4b3b1)
![image](https://github.com/Studio-Yandex-Practicum/adaptive_hockey_federation/assets/120438727/7b99f17b-1475-4038-bffe-aaba6f28b0e6)
![image](https://github.com/Studio-Yandex-Practicum/adaptive_hockey_federation/assets/120438727/efd86755-1c00-4249-89cd-493bdcb47369)

**Comment by KonstantinRaikhert** [comment]
@ViacheslavMelnikov Сам возьмешь?

**Comment by ViacheslavMelnikov** [comment]
Да, возьму. 19.04.2024, 03:20, "Konstantin Raikhert" ***@***.***>:
@ViacheslavMelnikov Сам возьмешь?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
-- С уважением Мельников Вячеслав Александрович
